.github/workflows/publish.yml

```
@@ -0,0 +1,51 @@
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '14.x'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install dependencies and build 🔧
+        run: npm install && npm run build
```

Let's use `npm ci` so it will strictly adhere to the lockfile.

```suggestion
        run: npm ci && npm run build
```